packages/helpers/constants/index.js

```diff
@@ -0,0 +1,8 @@
+const DYNAMIC_TEMPLATE_CHAR_WARNING = `
+Content with characters ', " or & may need to be escaped with three brackets
+{{{ content }}}
+See https://sendgrid.com/docs/ui/sending-email/using-handlebars/ for more information.`;
```

Comment:
url leads to -> 404 not found page

Reply:
Fixed by https://github.com/sendgrid/sendgrid-nodejs/pull/943